pennylane/measurements.py

```diff
@@ -494,7 +493,7 @@ def circuit(x):
     Raises:
         QuantumFunctionError: `op` is not an instance of :class:`~.Observable`
     """
-    if not isinstance(op, (Observable, qml.Hamiltonian)):
+    if not isinstance(op, (qml.operation.Observable, qml.Hamiltonian)):
```

Comment:
[should this be `qml.measurements.Observable`?]

Reply:
otherwise, would it cause the user to see warnings when using PL?

Reply:
I'm confused...

`Observable` is defined in `qml.operation`, not in `measurements`.

Reply:
Only the `ObservableReturnType` members were moved.

Reply:
Yep, should be good as is, `Observable` was not moved from `operation` to `measurements`. Rather the `ObservableReturnTypes` were (like `Sample`, `Variance`, etc.)

Reply:
Sorry guys, I shouldn't reply to email github notifications 😆 You are both 100% correct.

Reply:
No worries :)